rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCreateAlarmNode.java

```diff
@@ -49,29 +52,55 @@
 )
 public class TbCreateAlarmNode extends TbAbstractAlarmNode<TbCreateAlarmNodeConfiguration> {
 
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
I believe you can use static ObjectMapper: https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field